sentry-android-core/build.gradle.kts

```diff
@@ -49,6 +49,7 @@ android {
 
         // We run a full lint analysis as build part in CI, so skip vital checks for assemble tasks.
         checkReleaseBuilds = false
+        disable += "LogNotTimber"
```

Comment:
Wondering why we'd need that.

Reply:
Cause we use android.util.Log in the AndroidLogger and since we add Timber dependency now (compileOnly), it ships with lint rules that are complaining about that. I didn't find a way to exclude lint rules from the dependency

Reply:
I see, can we disable it from the sample App instead?

Reply:
But it's not a sample app, it's what we use to log SDK messages on Android https://github.com/getsentry/sentry-java/blob/main/sentry-android-core%2Fsrc%2Fmain%2Fjava%2Fio%2Fsentry%2Fandroid%2Fcore%2FAndroidLogger.java

Reply:
Ah ok, my bad.